core/domain/config_services_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+from core.domain import config_domain
+from core.domain import config_services
+from core.tests import test_utils
+
+
+class ConfigServicesDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ConfigServicesDomainUnitTests, self).setUp()
+        self.committer_id = u'configServicesCommitterID'
+        self.invalid_property_name = u'_no_such_property_name'
+        self.property_dummy_value = u'TEST_VALUE'
+        self.schema_unicode = 'unicode'
```

Comment:
You don't need to latch this onto `self`. In general carry around as little context as possible. In this case, just inline 'unicode' directly in line 35.